minecraft.js

```diff
@@ -66,6 +66,25 @@ app.post('/command', (req, res) => {
   }, 250);
 });
 
+app.get('/properties', (req, res) => {
+  const text = fs.readFileSync('./server.properties').toString('utf-8');
+  const result = fileToJSON(text);
+  res.send(result);
+});
+
+app.put('/properties', (req, res) => {
+  const properties = req.body;
+  let string = '#Minecraft server properties\n#(last boot timestamp)\n';
+  for (const property in properties) {
+    const sub_string = `${property}=${properties[property]}\n`;
+    string = string.concat(sub_string);
+  }
+  fs.writeFile('./server.properties', string, function (err) {
```

Comment:
You should use the async version of this as well so you can await it and then return an error if the save fails.

Reply:
I've made this change here as well!